File: Medium-Questions/01-even-Or-odd-(Fa)/C/one.c
#include <stdio.h>

int main(void)
{   

    int user;
    // get input from user
    printf("Enter a Number: ");
    scanf("%i", &user);

    // check for number between 1 to 100
    if (user > 0 && user < 100)
    {
        // check for Number is even or odd
        if (user % 2 == 0)
        {
            printf("it's Even Number :) ");
            return 0;
        }
        else
        {
            printf("Its odd Number :) ");
            return 0;
        }
    }
    // if number is out of Range
    else
    {
        printf("Invalid Number - Number must Between 1 to 100");
        return 0;
    }

}

File: Medium-Questions/01-even-Or-odd-(Fa)/python/One.py
import sys

# defind a function for decied a Number
# is Even or odd

def is_valid(number):
    """ this Function take one Argument and if Number is Even Return true
    otherwise Return False"""
    if (number % 2 == 0):
        return True
    return False    

# get input from user
input_user = int(input("Enter Number: "))

# check for numbre between 1 to 100
if (input_user > 0 and input_user <= 100):
    
    # pass number to is_valid function for checking
    if (is_valid(input_user)):
        print("Even Number (valid)")
        sys.exit(0)
    else:
        print("Odd Number (Not valid)")    
        sys.exit(0)
else:
    print("input out of Range 1 to 100")


sys.exit(0)
